src/wasm-builder.h

```diff
@@ -84,7 +84,7 @@ class Builder {
     auto* export_ = new Export();
     export_->name = name;
     export_->value = value;
-    export_->kind = ExternalKind::Function;
```

Comment:
This bugfix seems like it would have a bigger effect.

Reply:
Yeah, I think what happened is the code was copy-pasted from one place that did this (for functions). It wasn't used in another place til now...